python/llm/example/GPU/Lookahead/llama2/README.md

```diff
@@ -0,0 +1,69 @@
+# LLaMA2
+In this directory, you will find examples on how you could run LLaMA2 sym-int4 inference with lookahead optimization using IPEX-LLM on [Intel GPUs](../README.md). For illustration purposes, we utilize the [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) and [meta-llama/Llama-2-13b-chat-hf](https://huggingface.co/meta-llama/Llama-2-13b-chat-hf) as reference Llama2 models.
```

Comment:
maybe just `LLaMA2 inference with lookahead optimization` as lookahead optimization can be used for other precision too.